kaos/sensors/sensor.py

```diff
@@ -0,0 +1,86 @@
+"""
+This file defines a Sensor class. It represents a sensor object.
+
+Author: Team KMC-70.
+"""
+
+from flask import Blueprint
+
+class Sensor(object):
+    """
+    A Sensor object which represents a real-world sensor on a satellite.
+    The attributes here are not set in stone and can change according to
+    changes in the client's sensor models.
+
+    TODO: confirm these attribute descriptions. Most attribute descriptions can be found here:
+    http://help.agi.com/stk/index.htm#stk/ObjectMap_Sensor.htm%3FTocPath%3DGetting%2520Started%7CBuild%2520a%2520Scenario%7CAdd%2520STK%2520Objects%7CObject%2520Properties%7CSensors%7C_____0
+
+    Attributes:
+        sensor_id: unique sensor ID specifying the type of sensor.
+        agile: TODO: what is this?
+        beam_mode_type: TODO: what is this?
+        center_boresight_pitch_angle: sensor pitch angle.
+        center_boresight_roll_angle: sensor roll angle
+        dollar_cost: cost of sensor in dollars. TODO: is this necessary?
+        maneuver_speed: speed of sensor maneuverability.
+        name: sensor name (manufacturer name)
+        platform_id:
+        pointing_conic_half_angle: half-angle of sensor's conic view
+        pointing_horizontal_half_angle: The angle from the boresight (Z) direction to the
+                                        edge of the sensor in the XZ plane of the sensor's coordinate system.
+        pointing_vertical_half_angle: The angle from the boresight (Z) direction to the
+                                        edge of the sensor in the YZ plane of the sensor's coordinate system.
+        scene_duration: duration of viewing scene.
+        sensor_pointing_type: pointing type of the sensor. E.g. 3D model, along vector, fixed etc.
+        sensor_type: type of field of view of the sensor. E.g. Complex conic, custom, rectangular etc.
+        stabilization_time:
+        view_horizontal_half_angle: The angle from the boresight (Z) direction to the
+                                    edge of the sensor in the XZ plane of the sensor's coordinate system.
+        view_vertical_half_angle: The angle from the boresight (Z) direction to the
+                                    edge of the sensor in the YZ plane of the sensor's coordinate system.
+
+    Example attribute string (Note that the attributes of this string are out of order with the above):
+
+    '831cf482-b37b-4468-83cd-659814256892',true,,'0','0','100','1',
+    'WV110','41fe7f26-c557-49eb-aec8-dd658966f3a6','40.6','-1','-1','0',0,1,'1','2','0.6'
+    """
+    def __init__(self,
```

Comment:
@KMC-70/kaos-overlords Since the sensors seem to be provided in JSON, I'm wondering if we should just use the json library to parse them automatically. Would save time vs having to instantiate with almost 20 different fields) and is just as easy to access. Comments?

Reply:
Couple of comments on that:
1) Sensor data is actually provided in their PSQL database not a database. 

2) Having a constructor with fields is fine. This will go away once we subclass `flask.db.Model`. It will create the default constructor for us that takes all these fields as optional parameters.

3) The json thing is separate discussion, we have to create a serializer/deserializer to automatically handle this conversion between json text and python object representations of the database records.

TLDR: This is fine for now

Reply:
In that case, I'd suggest we leave the sensors off until the db is set up.

This file would be good as a README, though, to document what sensor models ought to contain.